src/Configuration/test/SpringBootCore.Test/SpringBootHostBuilderExtensionsTest.cs

```diff
@@ -88,6 +89,24 @@ public void GenericHostConfiguresIConfiguration_Spring_Application_Json()
         Environment.SetEnvironmentVariable("SPRING_APPLICATION_JSON", string.Empty);
     }
 
+    [Fact]
+    public void GenericHostConfiguresIConfiguration_Spring_Application_Json_Works_With_Placeholder()
+    {
+        Environment.SetEnvironmentVariable("SPRING_APPLICATION_JSON", "{\"foo.bar\":\"value\"}");
```

Comment:
When applying this fix to main, please use `EnvironmentVariableScope` instead.

Reply:
"When applying this fix to main" as in when you take the action to also apply this fix in main branch